server/svix-server/src/v1/utils.rs

```diff
@@ -116,9 +182,8 @@ where
         let Query(value) = Query::<T>::from_request(req)
             .await
             .map_err(|err| HttpError::bad_request(None, Some(err.to_string())))?;
-        value.validate().map_err(|x| {
-            let message = format!("Input validation error: [{}]", x).replace('\n', ", ");
-            HttpError::unprocessable_entity(None, Some(message))
+        value.validate().map_err(|e| {
+            HttpError::unprocessable_entity(validation_errors(vec!["query".to_owned()], e))
```

Comment:
We probably want the field name here too, so: `["query", "since"]` as an example.

Reply:
NVM, I'm silly, all good.